### PR TITLE
Mass System

### PR DESCRIPTION
**Pull request by johnny-la**
#### Notable Additions:
* Absorbable light
* Mass (Growing larger/smaller)
* Player light toggle

### CODE REVIEW

**Comment by alisterdev** [comment]
since you rename the character class, rename the folder as well?

[18:33] 
also, if you dont want to pass a million of things to constructor, you can pass them directly to methods that need the objects

[18:33] 
especially in a case where object are only gonna be used within that method

[18:34] 
also, getters/setters/properties can be used

File: Assets/Scripts/Light/LightToggle.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    /// </summary>
+	private bool lightsEnabled = true;
+    
+    void Start()
+    {
+        ToggleLights(beginEnabled);
+    }
+	
+	// Update is called once per frame
+	void Update ()
```
Comment (stellarxo) [inline Assets/Scripts/Light/LightToggle.cs]: minor spacing fix --> update()

**Comment by stellarxo** [comment]
I think it can be merged and maybe refactored later (especially addition of getters and setters?)
Just resolve your conflicts/rebase @johnny-la

**Comment by alisterdev** [comment]
Agree with @stellarxo, resolve conflicts and we merge

**Comment by johnny-la** [comment]
Conflicts have been resolved @alisterdev @stellarxo

**Comment by alisterdev** [comment]
Good to go

**Comment by alisterdev** [comment]
Definitely requires refactoring. LightAttribute and children should not extend MonoBehaviour.
LightMassModifier if statement bad formating
All vars to be private with necessary properties and constructors

**Pull request merged**